### PR TITLE
Copy tweaks from Max and JKT

### PR DESCRIPTION
**Pull request by schlosser**
Homepage & endorsements copy tweaks.

### CODE REVIEW

**Comment by asciimike** [comment]
LGTM

**Comment by asciimike** [comment]
@igorkofman please merge at your leisure

File: src/screens/Endorsements/Endorsements.style.js [hunk 1]
```diff
@@ -4,9 +4,8 @@ import { COLORS } from 'enums';
 
 export const Wrapper = styled.div`
   background-color: ${COLORS.LIGHTGRAY};
-  margin: -32px;
+  margin: -32px -32px 0;
```
Comment (goldblatt) [inline src/screens/Endorsements/Endorsements.style.js:7]: sigh

File: src/screens/Endorsements/Endorsements.style.js [hunk 2]
```diff
@@ -4,9 +4,8 @@ import { COLORS } from 'enums';
...
   padding: 1rem;
-  min-height: calc(100vh - 64px);
```
Comment (goldblatt) [inline src/screens/Endorsements/Endorsements.style.js:9]: did you confirm on a phone?

**Pull request merged**